python/pyarrow/tests/test_compute.py

```diff
@@ -3108,3 +3109,11 @@ def test_list_slice_bad_parameters():
         pc.list_slice(arr, 0, 1, step=0)
     with pytest.raises(pa.ArrowInvalid, match=msg + "-1"):
         pc.list_slice(arr, 0, 1, step=-1)
+
+
+def test_run_end_encode():
+    arr = pa.array([1, 1, 1, 2, 2, 1, 1, 1, 1, 1, 3, 3, 3, 3, 3, 3, 3, 3, 3])
+    encoded = pc.run_end_encode(arr, options=pc.RunEndEncodeOptions(pa.int64()))
```

Comment:
So this actually fails at the moment, because we can't yet put a REE C++ array in a pyarrow.Array object (`pyarrow_wrap_array` fails here)

On the short term, we can maybe put it just in the base class Array (I don't know if there is already an issue about adding proper python bindings?)

Reply:
Ooops. And I've ran the tests locally, but only now saw the lone F at the end of the green dots :D

I will fix this properly.